client/app/visualizations/chart/plotly/utils.js

```diff
@@ -69,9 +81,25 @@ function getFontColor(bgcolor) {
   return result;
 }
 
-export function normalizeValue(value) {
+function getPieHoverInfoPattern(options) {
+  const hasX = /{{\s*@@x\s*}}/.test(options.textFormat);
+  let result = 'text';
+  if (!hasX) result += '+label';
+  return result;
+}
+
+function getHoverInfoPattern(options) {
+  const hasX = /{{\s*@@x\s*}}/.test(options.textFormat);
+  const hasName = /{{\s*@@name\s*}}/.test(options.textFormat);
+  let result = 'text';
+  if (!hasX) result += '+x';
+  if (!hasName) result += '+name';
+  return result;
+}
+
+export function normalizeValue(value, dateTimeFormat = 'YYYY-MM-DD HH:mm:ss') {
   if (moment.isMoment(value)) {
-    return value.format('YYYY-MM-DD HH:mm:ss');
+    return value.format(dateTimeFormat);
```

Comment:
I'm not sure this is right. Plotly has its own rules on how to format date/time values on the axes (depends on the time range).

If we pass it a different format, it might not detect these values as datetime values. 

Did you try this?

Reply:
Maybe remove this setting for now and address it if we find the need in the future?

Reply:
I'll check if I can use it only for tooltips, if not - I'll remove it